packaged.yaml

```diff
@@ -15,10 +15,10 @@ Metadata:
     - atlas
     - serverless
     - MongoDBAtlas
-    LicenseUrl: s3://appdevstore/MongoDBAtlas/v1.0.13/5122657d5b9a0d3713e24d3a33eae431
+    LicenseUrl: s3://appdevstore/MongoDBAtlas/v1.0.15/5122657d5b9a0d3713e24d3a33eae431
```

Comment:
Why is this version different from inside VERSION? If you need to test why can't we use a fourth number for a patch?

Reply:
the one in VERSION file is version for python onprem package the one in this file is SAM version both are different

Reply:
can we make them the same? It's weird. If need to, bump up the Python version.

Reply:
To me SAM and Python are just different packaging media, shouldn't be the version of the collection solution.

Reply:
collection solution differs in SAM you create AWS related sources there would be instances where yaml file changes but not the python code which is why both the version differs.